src/main/java/org/truffleruby/core/rope/Rope.java

```diff
@@ -161,8 +158,15 @@ public byte get(int index) {
         return getByteSlow(index);
     }
 
+    @Override
+    public String toString() {
+        // This is designed to not have any side effects - compare to getString
```

Comment:
We should add a note this is also highly inefficient, as it will flatten on every call.
I wish there would be a way to check that a debugger is attached/is calling toString()